pilot/pkg/proxy/envoy/v2/config.go

```diff
@@ -718,8 +735,8 @@ func buildInboundListeners(mesh *meshconfig.MeshConfig, node model.Proxy,
 				model.SortRouteRules(rules)
 				for _, config := range rules {
 					switch config.Spec.(type) {
-					case *routing.RouteRule:
-						rule := config.Spec.(*routing.RouteRule)
+					case *v1alpha1.RouteRule:
```

Comment:
Do we need v1alpha1 here in v2 land?

Reply:
Costin wanted it as an interim thing until we migrate mixer and bookinfo tests to v1alpha3.